src/include/denc.h

```diff
@@ -170,9 +170,9 @@ struct denc_traits {
 #define WRITE_RAW_DENC(type)						\
   template<>								\
   struct denc_traits<type> {						\
-    enum { supported = 2 };						\
-    enum { featured = false };						\
```

Comment:
2 has a special meaning separate from 1/true

// NOTE: set supported == 2 instead of true.  This prevents these from
// getting glued into the legacy encode/decode methods; the overhead
// of setting up a contiguous_appender etc is likely to be slower.